kernel/common/get_slice.rs

```diff
@@ -1,8 +1,6 @@
-use core::slice;
-use core::str;
 use collections::range::RangeArgument;
 use core::ops::Range;
-use core::cmp::{max, min};
+use core::cmp::min;
```

Comment:
I think it is because the kernel implementation and the userspace implementation is differs? They might not be sync'ed.